src/trigrams.py

```diff
@@ -12,7 +13,10 @@ def main(path: str, words_to_generate: int):
 
     for i in range(words_to_generate - 2):
         key = " ".join(generated_text_list[-2:])
-        generated_text_list.append(generate_next_word(key, trigram_dict))
+        try:
+            generated_text_list.append(generate_next_word(key, trigram_dict))
+        except KeyError:
+            continue
```

Comment:
Is this error handling actually the desired behavior ("if no list of possible "next words" exists in the `trigram_dict` for our current `key`, just continue and do nothing with the starter word") or is this just there to smooth things over/make the failure silent?

Reply:
```def create_trigram_dict(path: str):
    """Takes in a file path and outputs a trigram dictionary created from the file"""
    trigram_dict = {}
    with open(path) as book:
        words = book.read().split()
        for index, word in enumerate(words[:-2]):
            trigram_dict.setdefault(words[index] + " " + words[index+1], [])
            trigram_dict[words[index] + " " + words[index+1]].append(words[index + 2])
    return trigram_dict
```

Fix `enumerate` not referencing `word`, as discussed IRL